check-redis/lib/check-redis.go

```diff
@@ -151,6 +149,12 @@ func checkSlave(args []string) *checkers.Checker {
 	}
 	defer c.Close()
 
+	if role, ok := (*info)["role"]; !ok {
+		return checkers.Unknown("couldn't get role")
+	} else if role == "master" {
+		return checkers.Ok("redis is not slave")
```

Comment:
When we use `check-redis slave ...`, we want to check the slave status. This condition matches the case when we wrongly specify the master node for `--host` option.

So `return checkers.Unknown("redis is not slave")` is better because we can notice that we wrongly specify `--host` option.